run_testlist.py

```diff
@@ -17,266 +17,424 @@
 env['MPICH_DIR']  = "<path_to_mpich>" #e.g./scratch/POC/mpich
 env['OPENMPI_DIR']= "<path_to_openmpi>" #e.g./scratch/POC/openmpi
 
-slf_testlist = [{'testcase': 'st_1tomany_cli2srv_inf1',
-                 'nServer': [2, 4, 8, 16, 32, 64, 128, 256, 512],
-                 'nClient': [1, 1, 1, 1, 1, 1, 1, 1, 1],
-                 # timeout in minutes
-                 'timeout': [15, 15, 15, 15, 15, 15, 15, 15, 15],
-                 'ppc': 1,
-                 'inflight': 1,
-                 'enabled': 0
-                 },
-                {'testcase': 'st_1tomany_cli2srv_inf16',
-                 'nServer': [2, 4, 8, 16, 32, 64, 128, 256, 512],
-                 'nClient': [1, 1, 1, 1, 1, 1, 1, 1, 1],
-                 # timeout in minutes
-                 'timeout': [15, 15, 15, 15, 15, 15, 15, 15, 15],
-                 'ppc': 1,
-                 'inflight': 16,
-                 'enabled': 0
-                 }
-                ]
 
-ior_testlist = [{'testcase': 'ioreasy_1to4',
-                 'nServer': [2, 4, 8, 16],
-                 'nClient': [8, 16, 32, 64],
-                 # timeout in minutes
-                 'timeout': [15, 15, 15, 15],
+self_testlist = [{'testcase': 'st_1tomany_cli2srv_inf1',
+                  # Number of servers, number of clients, timeout in minutes
+                  'testlist': [
```

Comment:
Sorry for the inconvenience.
Let's rename testlist here to testvariants.
So basically, i'd like to have:
testlist -> testcase -> testvariant